src/API/REST/V3/Routes/Donors/DonorStatisticsController.php

```diff
@@ -91,7 +91,13 @@ public function get_item($request)
                 'lifetimeAmount' => $query->getLifetimeDonationsAmount(),
                 'highestAmount' => $query->getHighestDonationAmount(),
                 'averageAmount' => $query->getAverageDonationAmount(),
+                'firstDonation' => $query->getFirstDonation(),
+                'donationCount' => $query->getDonationsCount(),
+                'lastContribution' => $query->getLastContribution()
             ],
+            'donorSince' => $donor->createdAt->format('Y-m-d H:i:s'),
+            'donorType' => $query->getDonorType()->label(),
```

Comment:
I also noticed getDonorType can return null which could potentially cause a fatal error if you try to call label() on null.

Reply:
Resolved: 283eeb4cdd32947359e0d74c0090775f7fb22a45